desmume/src/frontend/posix/shared/mic_openal.cpp

```diff
@@ -18,8 +18,13 @@
 
 #ifndef WIN32
 
-#include <AL/al.h>
-#include <AL/alc.h>
+#ifdef __APPLE__
+  #include <OpenAL/al.h>
+  #include <OpenAL/alc.h>
```

Comment:
out of curiosity... why is this different on apple, and why did it work in the past ? is it system libs vs homebrew ?

Reply:
I don't think anyone has ever trie to build interface under macOS with `clang` before. Maybe it's different when compiling with `gcc` for some reason or the libs were also installed with Homebrew, but usually the OpenGL and OpenAL headers are under `OpenGL/` and `OpenAL/` respectively.